### ISSUE TITLE
Move sass compilation and js minification to server

### ISSUE THREAD

**Issue by kalilsn** [jeancochrane/chano4mayor#21]
Instead of having to make sure we build these before pushing, we should do it on the server. Then we can have the local grunt setup just concatenate and have more aggressive minification happen on deployment. This will also allow people to make changes to css and js without having the tools installed.

**Comment by jeancochrane** [comment]
Seems like a good place to do this would be in the `hooks/post-receive` file in git?

**Comment by kalilsn** [comment]
yep! just need to install sass and whatever else on the server and get it
all configured!
